### ISSUE TITLE
allow for the functions to work without oedema data?

### ISSUE THREAD

**Issue by ernestguevarra** [nutriverse/ipctools#18]
@tomaszaba, should we consider datasets that have MUAC but no oedema data?

If so, then we probably need to re-factor the functions to accept no oedema data and still perform the checks and the calculations.

**Comment by tomaszaba** [comment]
@ernestguevarra, yes in some assessments (mostly) oedema data goes missing. I agree that we should re-factor the functions.
